src/translate/translate.js

```diff
@@ -1186,6 +1186,15 @@ const nodeQuery = ({
   neo4jTypeArgs,
   _id
 }) => {
+  const isCount = returnVariableName === '_Neo4jCount';
+  const variableName = isCount
+    ? resolveInfo.fieldName.substr(5)
+    : returnVariableName;
+  const typeName = isCount ? resolveInfo.fieldName.substr(5) : returnTypeName;
+  const schemaType = isCount
+    ? resolveInfo.schema.getType(variableName)
+    : returnSchemaType;
```

Comment:
Suggestion: extract this logic, as well as the name definitions (`${Neo4jTypeName}Count` + `Count${typeName}`) to some util file? Right now, as this is spread out across, it would be easy to forget updating this if we change the names?

Reply:
yes sir, its not clean still but its gonna get better with the group by